src/sde_default_alg.jl

```diff
@@ -13,16 +13,17 @@ function default_algorithm(
     end
 
     is_stiff = :stiff ∈ alg_hints
-    is_stratonovich = :Stratonovich ∈ alg_hints
+    is_stratonovich = :stratonovich ∈ alg_hints
```

Comment:
why this change?

Reply:
Because of docs at
https://github.com/SciML/DiffEqBase.jl/blob/13ac2da52c44494a57132245a3c6baec1c9cbb0e/src/solve.jl#L758